### PR TITLE
Material listbox

### PR DESCRIPTION
**Pull request by theOrlin**
Dealing with https://github.com/telerik/kendo-themes/issues/93

### CODE REVIEW

File: packages/material/scss/listbox/_theme.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@include exports( 'listbox/theme/material') {
+    $listbox-item-focused-bg: rgba(0, 0, 0, .07) !default;
+    $listbox-drag-clue-bg: $hovered-bg !default;
+
+    .k-listbox {
+        .k-list .k-item {
+            color: $base-text;
+
+            &.k-state-focused {
+                background-color: $listbox-item-focused-bg;
```
Comment (gyoshev) [inline packages/material/scss/listbox/_theme.scss]: In the Default theme, `.k-list .k-item` gets its styles from the `popup/_theme.scss` file, which is inherited by the Material theme. Do we need the additional styles?

**Comment by theOrlin** [comment]
The drag clue's `!important` width is needed because the jQuery widget sets the drag clue width inline, and the Material theme's drag clue does not have a preset width. The rest is a mistake caused by a copy-paste job from the Grid's drag clue override, where all the above properties are set inline.

File: packages/material/scss/popup/_theme.scss [hunk 2]
```diff
@@ -1 +1,17 @@
 @import "~@progress/kendo-theme-default/scss/popup/theme";
+
+@include exports( "list/theme/material") {
```
Comment (gyoshev) [inline packages/material/scss/popup/_theme.scss]: These styles are not needed. The focused state is added in the material/dropdowns branch, and everything else is already handled by the popup theme.

File: packages/default/scss/listbox/_layout.scss [hunk 3]
```diff
@@ -1,7 +1,12 @@
 @include exports( 'listbox/layout' ) {
     $listbox-margin: $padding-x !default;
+    $listbox-button-margin: $padding-x !default;
```
Comment (joneff) [inline packages/default/scss/listbox/_layout.scss]: By default, `$listbox-button-margin` should be equal to `$listbox-margin`

File: packages/default/scss/listbox/_theme.scss [hunk 4]
```diff
@@ -1,4 +1,7 @@
 @include exports('listbox/theme') {
+    $listbox-drag-clue-bg: $list-item-selected-bg !default;
+    $listbox-text: $list-item-selected-text !default;
```
Comment (joneff) [inline packages/default/scss/listbox/_theme.scss]: The variable name is misleading. I would expect to change the text color of the listbox widget, not the dragclue.

File: tests/visual/listbox.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                            <span class="k-icon k-i-x"></span>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="k-list-scroller k-selectable" data-role="selectable">
+                <ul class="k-reset k-list" role="listbox" tabindex="0">
+                    <li class="k-item" role="option" aria-selected="false" data-uid="6bd99714-38a2-4053-80c4-48494c8f6afb">Item</li>
+                    <li class="k-item k-state-selected" role="option" aria-selected="false" data-uid="19d76c92-35e9-4e0a-9135-f03d49dfe159">Selected</li>
+                    <li class="k-item k-state-focused" role="option" aria-selected="false" data-uid="a5a173ed-8c8e-488c-b297-faa2f68518f0">Focused</li>
```
Comment (joneff) [inline tests/visual/listbox.html]: The test does not show hovered, hover-selected and disabled item.

Additionally, there is not drag clue.

File: tests/visual/listbox.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                        <a href="#" class="k-button k-button-icon" data-command="transferAllFrom" title="Transfer All From" aria-label="Transfer All From"
+                            role="button">
+                            <span class="k-icon k-i-arrow-double-60-left"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="remove" title="Delete" aria-label="Delete" role="button">
+                            <span class="k-icon k-i-x"></span>
+                        </a>
+                    </li>
```
Comment (joneff) [inline tests/visual/listbox.html]: While the buttons do inherit styles, showing hovered, focused, pressed and disabled state will be a nice addition.

**Pull request merged**